<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="SmokeTest" duration-ms="57" started-at="2015-05-28T12:25:55Z" finished-at="2015-05-28T12:25:55Z">
    <groups>
      <group name="checkouttest">
        <method signature="CheckoutTest.checkoutTest()[pri:0, instance:CheckoutTest@31dc2ffd]" name="checkoutTest" class="CheckoutTest"/>
      </group> <!-- checkouttest -->
    </groups>
    <test name="TestSet_Firefox" duration-ms="57" started-at="2015-05-28T12:25:55Z" finished-at="2015-05-28T12:25:55Z">
      <class name="CheckoutTest">
        <test-method status="FAIL" signature="checkoutTest()[pri:0, instance:CheckoutTest@31dc2ffd]" name="checkoutTest" duration-ms="42" started-at="2015-05-28T09:25:55Z" finished-at="2015-05-28T09:25:55Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at org.openqa.selenium.interactions.Actions.<init>(Actions.java:41)
	at pages.HomePage.openSubcategoryBathing(HomePage.java:39)
	at CheckoutTest.checkoutTest(CheckoutTest.java:77)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:46)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:37)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkoutTest -->
        <test-method status="FAIL" signature="checkoutTest()[pri:0, instance:CheckoutTest@31dc2ffd]" name="checkoutTest" duration-ms="0" started-at="2015-05-28T09:25:55Z" finished-at="2015-05-28T09:25:55Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at org.openqa.selenium.interactions.Actions.<init>(Actions.java:41)
	at pages.HomePage.openSubcategoryBathing(HomePage.java:39)
	at CheckoutTest.checkoutTest(CheckoutTest.java:77)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:46)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:37)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkoutTest -->
      </class> <!-- CheckoutTest -->
    </test> <!-- TestSet_Firefox -->
  </suite> <!-- SmokeTest -->
</testng-results>
